src/components/WorkForm.js

```diff
@@ -358,34 +436,39 @@ const WorkForm = () => {
           return value && value[0] && value[0].size <= 2097152
         }).test("type",t("workWithUs_workForm_schemaYup_curriculum3"),
           function (value) {
-            return value && value[0] && value[0].type === "application/pdf"
+            return value && value[0] && (
+              value[0].type === 'image/jpg' ||
```

Comment:
```suggestion
              ['image/jpg', 'image/jpeg', 'application/pdf', 'application/msword'].includes(value[0].type)
```

A more readable way.

Reply:
I didn't know that way, gentleman